test/test-filter-process.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+. "test/testlib.sh"
+
+if [ "$GIT_LFS_USE_LEGACY_FILTER" == "1" ]; then
+  echo "skip: $0 (filter process disabled)"
+  exit
+fi
```

Comment:
Yeah, I think this also covers the case where the filter protocol is implemented in the installed version of Git, but we're not running the tests.